test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/TestAutoCreateSharedArchiveNoDefaultArchive.java

```diff
@@ -102,12 +102,10 @@ public static void main(String[] args) throws Exception {
             }
         }
 
-        // Now rename classes.jsa to old-classes.jsa
-        String dstDir = java_home_dst + File.separator + "lib" + File.separator + "server";
-        CDSTestUtils.rename(new File(dstDir + File.separator +  "classes.jsa"),
-                            new File(dstDir + File.separator +  "old-classes.jsa"));
-        System.out.println("======= renamed classes.jsa to old-classes.jsa");
-
+        // Remove all possible default archives
+        removeDefaultArchives(java_home_dst, "zero");
+        removeDefaultArchives(java_home_dst, "server");
+        removeDefaultArchives(java_home_dst, "zero");
```

Comment:
What is the point of doing "zero" part twice?

Reply:
Fixed. It should be "client".